src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -279,6 +288,16 @@ def build_model(self,  # noqa: C901 - too complex
                                                (ie: path/to/checkpoint.pth). If provided, will automatically attempt to
                                                load the checkpoint even if the load_checkpoint flag is not provided.
 
+        Additional key:values that can be passed through arch_params:
```

Comment:
qat params are training params, not arch_params

Reply:
Passed through training_params and instantiated as a phase callback behind the scenes now.